### PR TITLE
Create Nav component, extract svgs to the assets folder, update height to hug content

### PR DESCRIPTION
**Pull request by Stevensauro**

### CODE REVIEW

File: fetools-app/src/components/Footer.jsx [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
+            max-sm:flex-col max-sm:items-center
             ">
                 <p className="
                 font-bold block max-w-fit min-w-fit 
                 max-sm:tracking-tight
                 ">Site Copyright &copy; 2024 {appName}.</p>
                 <ul className="flex flex-row gap-4">
                     <li>
-                        <Link to="/" className="
+                        <Link to="/Team" className="
```
Comment (joekotvas) [inline fetools-app/src/components/Footer.jsx:36]: Slugs should be lower cased.

File: fetools-app/src/components/Header.jsx [hunk 2]
```diff
@@ -1,7 +1,12 @@
 import { useState} from 'react';
-import {Link } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 import { appName } from "../pages/data"
+
 import MobileMenu from './MobileMenu'
+import closeIcon from '../assets/x-solid.svg'
+import hambuerIcon from '../assets/hamburger.svg'
```
Comment (joekotvas) [inline fetools-app/src/components/Header.jsx:7]: hamburgerIcon 😆

File: fetools-app/src/components/Header.jsx [hunk 3]
```diff
@@ -88,24 +45,15 @@ export default function Header(){
...
             return(
-                <svg id='hamburguer-btn' onClick={()=>setClick(!click)} width="39" height="39" viewBox="0 0 39 39" fill="none" xmlns="http://www.w3.org/2000/svg"
-                className="
-                cursor-pointer flex-2 fill-current self-center
-                sm:hidden
-                ">
-                    <path d="M4.875 27.625H34.125M4.875 19.5H34.125M4.875 11.375H34.125" stroke="black" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"/>
-                </svg>
+                <img onClick={()=>setClick(!click)}
+                src={hambuerIcon} alt='hamburger icon'
```
Comment (joekotvas) [inline fetools-app/src/components/Header.jsx:49]: hamburgerIcon variable name will need to be updated here also

**Review by joekotvas — APPROVED** [review]
Great! I've added a few observations.

**Comment by zizi1404** [comment]
[Epic: Landing Page](https://trello.com/c/f61NL2r7/6-epic-landing-page)

**Comment by zizi1404** [comment]
[Epic: Landing Page](https://trello.com/c/f61NL2r7/6-epic-landing-page)

**Pull request merged**